From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: x86/sev: Use TSC_FACTOR for Secure TSC frequency calculation
Patch-mainline: never, kabi
References: git-fixes

Commit 52e1a03e6cf6 (x86/sev: Use TSC_FACTOR for Secure TSC frequency
calculation) changed 4 reserved bytes to an u32. This makes the kabi
checker to complain.

It's just a cosmetic change from the kabi POV. Just hide it.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 arch/x86/include/asm/sev.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/x86/include/asm/sev.h
+++ b/arch/x86/include/asm/sev.h
@@ -263,11 +263,15 @@ struct snp_secrets_page {
 	u8 svsm_guest_vmpl;
 	u8 rsvd3[3];
 
+#ifdef __GENKSYMS__
+	u8 rsvd4[3744];
+#else
 	/* The percentage decrease from nominal to mean TSC frequency. */
 	u32 tsc_factor;
 
 	/* Remainder of page */
 	u8 rsvd4[3740];
+#endif
 } __packed;
 
 struct snp_msg_desc {
